praw/models/reddit/collections.py

```diff
@@ -471,6 +475,7 @@ def create(self, title: str, description: str):
                 "sr_fullname": self.subreddit_fullname,
                 "title": title,
                 "description": description,
+                "display_layout": display_layout,
```

Comment:
Let's only include this in the data if `display_layout` is not `None`.

Reply:
Why'd you edit out the "not" `None`? Shouldn't we include the data if it's only not `None`?

Reply:
Oh yep, you're right, for some reason I read that wrong.